### ISSUE TITLE
Attempting to focus a window with no title fails

### ISSUE THREAD

**Issue by b0o** [korreman/sway-overfocus#10]
If the target window has no title, sway-overfocus fails.

```
$ sway-overfocus split-lw float-lw
error: no valid focus command
```

The window that should be the target (notice name is `null`):

```json
{
  "id": 63,
  "type": "con",
  "orientation": "none",
  "percent": 0.3319633259563705,
  "urgent": false,
  "marks": [],
  "focused": true,
  "layout": "none",
  "border": "pixel",
  "current_border_width": 4,
  "rect": {
    "x": 2154,
    "y": 18,
    "width": 1050,
    "height": 1668
  },
  "deco_rect": {
    "x": 0,
    "y": 0,
    "width": 0,
    "height": 0
  },
  "window_rect": {
    "x": 4,
    "y": 4,
    "width": 1042,
    "height": 1660
  },
  "geometry": {
    "x": 0,
    "y": 0,
    "width": 500,
    "height": 500
  },
  "name": null,
  "window": 6291457,
  "nodes": [],
  "floating_nodes": [],
  "focus": [],
  "fullscreen_mode": 0,
  "sticky": false,
  "pid": 3670786,
  "app_id": null,
  "visible": true,
  "max_render_time": 0,
  "shell": "xwayland",
  "inhibit_idle": false,
  "idle_inhibitors": {
    "user": "none",
    "application": "none"
  },
  "window_properties": {
    "transient_for": null
  }
}
```

If I forcefully set a name with xprop, sway-overfocus works as expected:

```
$ xprop -id 6291457 -set WM_NAME test
$ sway-overfocus split-lw float-lw
[
  {
    "success": true
  }
]
```

**Comment by korreman** [comment]
Thanks for catching this. `focus_command` was checking for a name even when it wasn't needed. This should be fixed by 9113bea. I'm having a bit of trouble creating a nameless window, so I would appreciate if you could test it.

**Comment by b0o** [comment]
Yes, that fixed the issue! Thanks for the super quick response.
